cargo-process.el

```diff
@@ -377,6 +379,15 @@ MANIFEST (i.e. Cargo.toml)."
           (end-of-line))
         deps-list))))
 
+(defun cargo-process--augment-cmd-for-os (opens-external cmd)
+  "Augment the cargo CMD according to OS. OPENS-EXTERNAL is non-nil
+if the CMD is expected to open and external application."
+  (if (and opens-external (and
```

Comment:
`and` takes a `&rest` argument, meaning that
``` Emacs Lisp
(and t 
     (and t 
          nil))
```

Is equivalent to

``` Emacs Lisp
(and t
     t
     nil)
```

Might as well use a single `and` here